go/protocol/keybase1/extras.go

```diff
@@ -1774,6 +1786,61 @@ func (t TeamInviteType) String() (string, error) {
 	return "", nil
 }
 
+func TeamInviteTypeEq(a, b TeamInviteType) bool {
```

Comment:
Make this `TeamInviteType.Eq` or `.Equal` for consistency with other stuff?

Reply:
👍